recipes/xeus-octave/build.sh

```diff
@@ -0,0 +1,6 @@
+
+#!/bin/bash
+
+cmake ${CMAKE_ARGS}
```

Comment:
Use ninja to make use of all CPUs (better)

```suggestion
cmake -GNinja ${CMAKE_ARGS}
```